packages/markers/marker.go

```diff
@@ -156,18 +156,19 @@ func (m *Markers) SequenceIDs() (sequenceIDs SequenceIDs) {
 	return NewSequenceIDs(sequenceIDsSlice...)
 }
 
-// FirstMarker returns the first Marker in the collection. It can for example be used to retrieve the new Marker that
-// was assigned when increasing the Index of a Sequence.
-func (m *Markers) FirstMarker() (firstMarker *Marker) {
+// HighestSequenceMarker returns the Marker of the highest SequenceID in the collection. It can for example be used to
+// retrieve the new Marker that was assigned when increasing the Index of a Sequence.
```

Comment:
It returns the Marker of the Sequence that is furthest away from the root of the Sequence DAG or in other words the marker of the Sequence with the highest SequenceID.

Since SequenceIDs are monotonically increasing it can not happen that a later Sequence has a lower index than one of its ancestors.

Reply:
do you mean lower marker index or lower sequence rank (index) ? because I feel it can happen that in the list of parent markers there can be a marker A with higher sequenceID than marker B, but B has a higher marker index than A

Reply:
I mean that the Sequence that has a higher SequenceRank than another Sequence will also always have a higher SequenceID than that other Sequence.

Reply:
thanks finally got it